.github/workflows/ghcr.yml

```diff
@@ -15,7 +15,7 @@ env:
 
 jobs:
   build-and-push:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Should we pin this to `24.04` which is the current latest? In October latest might change to 25.10 which could cause unexpected problems.

Reply:
Done!